Microsoft.Azure.Cosmos.Samples/Tools/FaultInjection/src/FaultInjectionServerErrorResultBuilder.cs

```diff
@@ -42,15 +42,17 @@ public FaultInjectionServerErrorResultBuilder WithTimes(int times)
         /// 
         /// Only used RESPONSE_DELAY and CONNECTION_DELAY.
         /// 
-        /// For RESPONSE_DELAY, it is the delay added before the response.
+        /// For SEND_Delay, it is the delay added before the request is sent.
```

Comment:
If these are public enums then prefer

```suggestion
        /// For <see cref="FaultInjectionServerErrorType.SendDelay"/>, it is the delay added before the request is sent.
```

The same with the other comments. `RESPONSE_DELAY` for example does not really represent any of the enum values, it does not match the casing nor name (if we wanted to at least match the name it would be ResponseDelay)